hack/windows/Prepare-Node.ps1

```diff
@@ -82,21 +90,24 @@ mkdir -force C:\var\log\kubelet
 mkdir -force C:\var\lib\kubelet\etc\kubernetes
 mkdir -force C:\etc\kubernetes\pki
 New-Item -path C:\var\lib\kubelet\etc\kubernetes\pki -type SymbolicLink -value C:\etc\kubernetes\pki\
-
+{{CONTAINER_RUNTIME}}
 $StartKubeletFileContent = '$FileContent = Get-Content -Path "/var/lib/kubelet/kubeadm-flags.env"
-$global:KubeletArgs = $FileContent.Trim("KUBELET_KUBEADM_ARGS=`"")
+$global:KubeletArgs = $FileContent.TrimStart(''KUBELET_KUBEADM_ARGS='').Trim(''"'')
+
+$global:containerRuntime = 
 
-$netId = docker network ls -f name=host --format "{{ .ID }}"
+if ($global:containerRuntime -eq "Docker") {
```

Comment:
I don't get this. It looks to me L97 reset the variable and L99 checks whether it equals to `Docker`, how could it ever get true?